packages/docs/pages/Table/TablePage.tsx

```diff
@@ -182,6 +182,25 @@ const TablePage = () => {
         {/* jsx-to-string:end */}
       </CodePreview>
 
+      <H2>Usage with column header tooltip</H2>
+
+      <CodePreview>
+        {/* jsx-to-string:start */}
+        <Table
+          columns={[
+            { header: 'Sku', hash: 'sku', tooltip: 'Header tooltip', render: ({ sku }) => sku },
+            { header: 'Name', hash: 'name', render: ({ name }) => name },
+            { header: 'Stock', hash: 'stock', render: ({ stock }) => stock },
+          ]}
+          items={[
+            { sku: 'SM13', name: '[Sample] Smith Journal 13', stock: 25 },
+            { sku: 'DPB', name: '[Sample] Dustpan & Brush', stock: 34 },
+          ]}
+          stickyHeader
+        />
+        {/* jsx-to-string:end */}
+      </CodePreview>
+
```

Comment:
We could just add `tooltip` to the first example on the page instead of creating a new one. Seems pretty excessive. 🤔

Reply:
Done)